synfig-studio/src/gui/app.cpp

```diff
@@ -1673,67 +1677,13 @@ App::App(const synfig::String& basepath, int *argc, char ***argv):
 			splash_screen.show();
 		}
 
-		// Look for any files given on the command line,
-		// and load them if found.
-		for(;*argc>=1;(*argc)--)
-			if((*argv)[*argc] && (*argv)[*argc][0]!='-')
-			{
-				studio_init_cb.task(_("Loading files..."));
-				splash_screen.hide();
-				open((*argv)[*argc]);
-				opened_any = true;
-				splash_screen.show();
-			}
-
-		// if no file was specified to be opened, create a new document to help new users get started more easily
-		if (!opened_any && !getenv("SYNFIG_DISABLE_AUTOMATIC_DOCUMENT_CREATION"))
-			new_instance();
-
 		studio_init_cb.task(_("Done."));
 		studio_init_cb.amount_complete(10000,10000);
 
-		// To avoid problems with some window managers and gtk >= 2.18
-		// we should show dock dialogs after the settings load.
-		// If dock dialogs are shown before the settings are loaded,
-		// the windows manager can act over it.
-		// See discussions here:
-		// * https://synfig.org/forums/viewtopic.php?f=1&t=1131&st=0&sk=t&sd=a&start=30
-		// * https://synfig.org/forums/viewtopic.php?f=15&t=1062
-		dock_manager->show_all_dock_dialogs();
-
-		main_window->present();
-
 		splash_screen.hide();
 
-		String message;
-		String details;
-		/*
-		if (App::enable_experimental_features) {
-			message = _("Following experimental features are enabled: ");
-			message += ("Skeleton Layer");
-			detials = _("The experimental features are NOT intended for production use. "
-					"It is quite posiible their functionality will change in the "
-					"future versions, which can break compatibility for your "
-					"files. Use for testing purposes only. You can disable "
-					"experimental features on the \"Misc\" tab of Setup dialog.");
-		}
-		*/
-#ifdef _WIN32
-		if (message!=""){
-			message = _("There is a bug, which can cause computer to hang/freeze when "
-					"resizing the canvas window.");
-			details = _("If you got affected by this issue, consider pressing ALT+TAB "
-					"to unfreeze your system and get it back to the working "
-					"state. Please accept our apologies for inconvenience, we "
-					"hope to get this issue resolved in the future versions.");
-		}
-#endif
-		if (message!="")
-			dialog_message_1b("WARNING",
-					message,
-					details,
-					_("Got it"));
 	}
+
```

Comment:
?

Reply:
Will get removed in next commit ! Thanks for mentioning 🙂